ckanext/datarequests/plugin/flask_plugin.py

```diff
@@ -54,13 +54,13 @@ def get_blueprint(self):
                 ('GET', 'POST',),
             ),
             (
-                "/{}/follow/<datarequest_id>".format(constants.DATAREQUESTS_MAIN_PATH),
+                "/{}/follow/<id>".format(constants.DATAREQUESTS_MAIN_PATH),
```

Comment:
isn't id too generic, why did we need to shrink it?

Reply:
It has to match what's actually used in the page, otherwise there's an error. We could change it, but would then also need to change the Pylons routing, which just uses 'id'.